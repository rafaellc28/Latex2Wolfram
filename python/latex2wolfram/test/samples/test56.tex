% Test natural logarithm expression
\ln(1.2)
