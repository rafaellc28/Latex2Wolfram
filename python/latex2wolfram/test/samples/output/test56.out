ln(1.2)
